versions.tf

```diff
@@ -1,10 +1,18 @@
 terraform {
-  required_version = ">= 0.13.0"
+  required_version = "0.13.5"
```

Comment:
The versions in this file shouldn't be pinned. They should stay open ended with a `>=` since this is the module, not the impl

Reply:
This has been fixed.